asllib/doc/ASLTypingReference.tex

```diff
@@ -1882,65 +2228,65 @@ \subsection{Goal}
   Similarly, we determine the negation of integer constraints.
 
 \subsection{Prose}
-  \texttt{t} is the result of checking compatibility of a unary operator \texttt{op} with
-  type \texttt{t1} and one of the following applies:
+  $\vt$ is the result of checking compatibility of a unary operator \texttt{op} with
+  type \texttt{t1} and one of the following apply:
   \begin{itemize}
   \item All of the following apply:
     \begin{itemize}
       \item \texttt{op} is \texttt{BNOT};
       \item \texttt{t1} type-satisfies \texttt{boolean};
-      \item \texttt{t} is \texttt{boolean};
+      \item $\vt$ is \texttt{boolean};
     \end{itemize}
 
   \item All of the following apply:
   \begin{itemize}
     \item \texttt{op} is \texttt{NEG};
     \item \texttt{t1} type-satisfies \texttt{real};
-    \item \texttt{t} is \texttt{real};
+    \item $\vt$ is \texttt{real};
   \end{itemize}
 
   \item All of the following apply:
     \begin{itemize}
     \item \texttt{op} is \texttt{NEG};
-    \item One of the following applies:
+    \item One of the following apply:
       \begin{itemize}
       \item \texttt{t1} type-satisfies \texttt{integer};
       \item \texttt{t1} type-satisfies \texttt{real};
       \end{itemize}
-     \item One of the following applies:
+     \item One of the following apply:
        \begin{itemize}
        \item All of the following apply:
          \begin{itemize}
          \item \texttt{t1} has the structure of an unconstrained integer;
-         \item \texttt{t} is an unconstrained integer;
+         \item $\vt$ is an unconstrained integer;
          \end{itemize}
        \item All of the following apply:
          \begin{itemize}
          \item \texttt{t1} has the structure of a constrained integer;
-         \item \texttt{t} is a constrained integer whose constraint is ;
+         \item $\vt$ is a constrained integer whose constraint is ;
```

Comment:
missing constraint - please fill in

Reply:
this is messed up from the previous version. I'll get to it in the next commit.